drv/stm32xx-sys/src/main.rs

```diff
@@ -88,8 +126,97 @@ fn main() -> ! {
+                            // Safety: field modeled incorrectly in PAC
+                            2 => unsafe { w.exti2().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            _ => unsafe { w.exti3().bits(port as u8) },
+                        }),
+                        1 => syscfg.exticr2.modify(|_, w| match slot {
+                            // Safety: field modeled incorrectly in PAC
+                            0 => unsafe { w.exti4().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            1 => unsafe { w.exti5().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            2 => unsafe { w.exti6().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            _ => unsafe { w.exti7().bits(port as u8) },
+                        }),
+                        2 => syscfg.exticr3.modify(|_, w| match slot {
+                            // Safety: field modeled incorrectly in PAC
+                            0 => unsafe { w.exti8().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            1 => unsafe { w.exti9().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            2 => unsafe { w.exti10().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            _ => unsafe { w.exti11().bits(port as u8) },
+                        }),
+                        _ => syscfg.exticr4.modify(|_, w| match slot {
+                            // Safety: field modeled incorrectly in PAC
+                            0 => unsafe { w.exti12().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            1 => unsafe { w.exti13().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            2 => unsafe { w.exti14().bits(port as u8) },
+                            // Safety: field modeled incorrectly in PAC
+                            _ => unsafe { w.exti15().bits(port as u8) },
+                        }),
+                    }
+                }
+            }
+
+            // Grab a pointer to EXTI while we're here, as the server setup
+            // below will use it. (Do not let the apparent block scope of cfg_if
+            // fool you! This escapes to the code below and is not unused.)
+            //
+            // Safety: same complaint as above, this is needlessly unsafe in the
+            // API the way we use peripherals.
+            let exti = unsafe { &*device::EXTI::ptr() };
+
+            // exti and syscfg are used in server init below
```

Comment:
tiny comment nit: only `exti` is used in the server init

Reply:
Good catch, legacy of an earlier revision. Will fix. (In fact I may move the exti unsafe block down to where it's installed in the server field, to reduce its available scope.)